Added "Privacy Policy and Terms and Conditions" Pages

MastanSayyad opened the pull request:
## Fixes Issue
This Pr fixes the issue #92 


## Changes proposed
Hey @AbhiPatel10 I have Created Privacy policy and Terms and conditions pages and link to the footer

## Screenshots
**Footer**

![image](https://github.com/AbhiPatel10/openzone/assets/101971980/cbcb1842-edce-4f7b-a0fb-cdfe648079b4)

**Privacy Policy**

![image](https://github.com/AbhiPatel10/openzone/assets/101971980/e7ed5944-8f58-4ada-968e-216efca6ab59)

**Terms and Conditions**

![image](https://github.com/AbhiPatel10/openzone/assets/101971980/4206d242-e91c-4beb-989e-e0489a08dedf)

AbhiPatel10 commented on pnpm-lock.yaml:
Remove This file Changes

AbhiPatel10 reviewed (changes requested):
All Looks Good to me, Thanks for your contribution, Just remove The change from pnpm-lock file

MastanSayyad commented:
@AbhiPatel10 Hey I have removed the changes from the file. Mistakenly I deleted the whole file but later I copy pasted the pnpm file from your repository so it should be fine
